messaging/src/test/groovy/com/mx/path/connect/messaging/MessageResponseTest.groovy

```diff
@@ -66,7 +66,7 @@ class MessageResponseTest extends Specification {
     def json = subject.toJson()
 
     then:
-    json == "{\"messageHeaders\":{\"headers\":{\"arbitrary\":\"value\"},\"sessionId\":\"session123\"},\"status\":\"SUCCESS\"}"
+    json == "{\"messageHeaders\":{\"headers\":{\"arbitrary\":\"value\"},\"sessionId\":\"session123\"},\"status\":\"SUCCESS\",\"exception\":{}}"
```

Comment:
Seems that the json text in this test now has exception, seems fine to me. Fixing test.